bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -119,6 +119,11 @@ public class ClientLibrariesImpl implements ClientLibraries {
     @Nullable
     private String media;
 
+    @Inject
+    @Named(OPTION_ONLOAD_CSS)
```

Comment:
Do you expect to use this option alongside JS `onload` in the same clientlib call? If not, we can just use the existing option. If you need to have separate `onload` handlers for JS and CSS, you can just split the clientlib call in two.